virtual_rainforest/models/soil/soil_model.py

```diff
@@ -132,29 +147,122 @@ def spinup(self) -> None:
         """Placeholder function to spin up the soil model."""
 
     def update(self) -> None:
-        """Placeholder function to update the soil model."""
+        """Function to update the soil model."""
```

Comment:
This can be terser:

```suggestion
        """Update the soil model."""
```

I'd also say *how* you're updating the soil model.